hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -227,7 +259,130 @@ private long gasFeeInTinybars(final TransactionBody.Builder txBody, Instant cons
 	}
 
 	@Override
-	public Gas gasRequirement(final Bytes bytes) {
+	public Gas gasRequirement(final Bytes input) {
```

Comment:
This is undoing work in #2780.  We need to calculate gas costs as part of execution rather than as a function of the input bytes.  The call processor was adjusted to deal with this re-ordering.